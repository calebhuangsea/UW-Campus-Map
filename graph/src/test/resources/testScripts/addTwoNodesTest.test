# Test add two nodes file
# Tests that a graph can be created
# two nodes can be added to the graph

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

# Print the nodes in the graph
ListNodes graph1