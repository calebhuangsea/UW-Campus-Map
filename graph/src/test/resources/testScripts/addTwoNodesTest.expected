# Test add two nodes file
# Tests that a graph can be created
# two nodes can be added to the graph

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1
added node n2 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2